.thoth.yaml

```diff
@@ -1,12 +0,0 @@
-managers:
```

Comment:
If we are removing this.
feel free to remove this file as well https://github.com/opendatahub-io/opendatahub-operator/blob/master/version.py

Reply:
@LaVLaS  one additional notes, as configuration is removed
please uninstall `khebhut` bot of this repo, or else it would open a pr to add the config back.